vpr/src/place/move_utils.cpp

```diff
@@ -5,6 +5,11 @@
 
 #include "vtr_random.h"
 
+//Run-time flag to control when placer debug information is printed
+//Note : only enables debug output if compiled with VTR_ENABLE_DEBUG_LOGGING is defined
+#ifdef VTR_ENABLE_DEBUG_LOGGING
+bool f_placer_debug = false;
```

Comment:
Should measure f_placer_debug performance impact.  Does it control more than manual moves?  If so, probably separating manual moves from it would be a good idea, and if manual moves has no performance impact (when off) then I'd turn it on by default.

Reply:
I think by manual move you mean breakpoints. If so, yes f_placer_debug is only used to stop the program whenever a breakpoint is reached. But it shouldn't be true by default because setting it true would stop the program after each perturbation.

Reply:
OK, just comment the variable use then.